CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.354 Feb 12 2024
+- Add `AdditionalSecurityGroupIds` to `AWS Node Pool` type.
```

Comment:
Can we add this to a different commit, just to preserve the history :pray:

Reply:
Sure thing, split into two commits.